src/cmds/qsub.c

```diff
@@ -472,15 +472,29 @@ expand_varlist(char *varlist)
 	char *p1, *p2, *p;
 	char *ev;
 	int  v_value1_sz=0;
+	char *pc;
+	int special_char_cnt = 0;
+	int len = 0;
+
+	/*
+	 * count special characters as they are escaped with '\' in copy_env_value function
+	 * so that this is useful while calculating the accurate size of the destination string
```

Comment:
update the comment to mention that the length is also calculated in the below loop.

Reply:
Implemented